pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -980,14 +980,24 @@ func validateVSphere(m *Machine, clusterID string) (bool, []string, utilerrors.A
 
 	errs = append(errs, validateVSphereNetwork(providerSpec.Network, field.NewPath("providerSpec", "network"))...)
 
-	if providerSpec.NumCPUs < minVSphereCPU {
-		warnings = append(warnings, fmt.Sprintf("providerSpec.numCPUs: %d is less than the minimum value (%d): the minimum value will be used instead", providerSpec.NumCPUs, minVSphereCPU))
```

Comment:
Why are we removing `the minimum value will be used instead`?

Reply:
I completely forgot about this behavior.